### PR TITLE
added synonym

### PR DESCRIPTION
**Pull request by peterhan7**

### CODE REVIEW

File: src/app/study/[id]/view/page.tsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ export default async function StudyPage({ params }: { params: { id: string } })
     fetchPassageContent(studyId),
   ]);
 
+
```
Comment (teamsela) [inline src/app/study/[id]/view/page.tsx:31]: Can you remove the unnecessary change to this file?

File: src/components/StudyPane/InfoPane/Motif/index.tsx [hunk 2]
```diff
@@ -16,6 +16,22 @@ const Motif = ({
 
   const activeClasses = "text-primary border-primary";
   const inactiveClasses = "border-transparent";
+  let rootWordsMap = new Map<number, HebWord[]>();
```
Comment (teamsela) [inline src/components/StudyPane/InfoPane/Motif/index.tsx:19]: Can you move this block back into Root component? Just pass content into Root like before

File: src/components/StudyPane/InfoPane/index.tsx [hunk 3]
```diff
@@ -39,7 +39,7 @@ const InfoPane = ({
             {/* Conditionally render the content based on infoPaneAction */}
             <div className="mx-6 h-full">
                 {infoPaneAction === InfoPaneActionType.structure && <Structure />}
-                {infoPaneAction === InfoPaneActionType.motif && <Motif content={content} />}
+                {infoPaneAction === InfoPaneActionType.motif && <Motif content={content}/>}
```
Comment (teamsela) [inline src/components/StudyPane/InfoPane/index.tsx:42]: Remove unnecessary change

File: src/components/StudyPane/InfoPane/Motif/CategoryBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}) => {
+    const { ctxSetCategoryRoots } = useContext(FormatContext);
+
+    const handleClick = () => {
+        if(category == selectedCategory){
+            setSelectedCategory("");
+            ctxSetCategoryRoots([]);
+        }else{
+            setSelectedCategory(category);
+            ctxSetCategoryRoots(value.strongNumbers);
```
Comment (teamsela) [inline src/components/StudyPane/InfoPane/Motif/CategoryBlock.tsx:25]: Can we just use ctxSetSelectedHebWords? Root block also uses ctxSetSelectedHebWords

**Review by teamsela — CHANGES_REQUESTED** [review]
Proposed some changes

File: src/components/StudyPane/InfoPane/Motif/index.tsx
Comment (teamsela) [inline src/components/StudyPane/InfoPane/Motif/index.tsx:1]: Change the name of the tab from Synonyms to Categories.

File: src/xata.ts [hunk 5]
```diff
@@ -213,4 +213,4 @@ export const getXataClient = () => {
 
   instance = new XataClient();
   return instance;
-};
+};
```
[the remark this answers is not in the document]
Comment (peterhan7) [reply src/xata.ts:216]: I don't know what changed here

**Review by teamsela — APPROVED** [review]
